apache-couchdb-mixin/dashboards/couchdb-nodes.libsonnet

```diff
@@ -905,19 +905,9 @@ local bulkRequestsPanel = {
   },
 };
 
-local responseStatusOverviewPanel = {
+local responseStatusErrorOverviewPanel = {
   datasource: promDatasource,
   targets: [
-    prometheus.target(
-      'sum by(instance, cluster) (increase(couchdb_httpd_status_codes{' + matcher + ', code=~"2.*"}[$__interval])) != 0',
-      datasource=promDatasource,
-      legendFormat='{{cluster}} - {{instance}} - 2xx',
-    ),
-    prometheus.target(
-      'sum by(instance, cluster) (increase(couchdb_httpd_status_codes{' + matcher + ', code=~"3.*"}[$__interval])) != 0',
-      datasource=promDatasource,
-      legendFormat='{{cluster}} - {{instance}} - 3xx',
-    ),
     prometheus.target(
       'sum by(instance, cluster) (increase(couchdb_httpd_status_codes{' + matcher + ', code=~"4.*"}[$__interval])) != 0',
```

Comment:
Minor nitpick, but can we change the colours so 4xx isn't green? 😅 

Maybe yellow for 4xx and red for 5xx?

Reply:
Copying my message from slack:
> Hmm that is a good point, but I think here I don’t mind it being an overview style panel? Could also help reduce visual clutter a little, especially when scaled to more than a handful of instances
E.g. it shows the sum of responses across all instances by default, but let you drill down using the selectors above. I think if we did that we could put colours on there, but yeah agree if we want to include the instance label in there then specifying colours is too much.